specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/stable/2023-07-20/azurelargeinstance.json

```diff
@@ -0,0 +1,1295 @@
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Accepted",
+            "Creating",
+            "Updating",
+            "Failed",
+            "Succeeded",
+            "Deleting",
+            "Migrating",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "AzureLargeInstanceProvisioningStatesEnum",
+            "modelAsString": true
+          },
+          "description": "State of provisioning of the AzureLargeInstance"
+        }
+      },
+      "description": "Describes the properties of an Azure Large Instance."
+    },
+    "HardwareProfile": {
+      "type": "object",
+      "properties": {
+        "hardwareType": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Cisco_UCS",
+            "HPE",
+            "SDFLEX"
+          ],
+          "x-ms-enum": {
+            "name": "AzureLargeInstanceHardwareTypeNamesEnum",
+            "modelAsString": true
+          },
+          "description": "Name of the hardware type (vendor and/or their product name)"
+        },
+        "azureLargeInstanceSize": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "S72m",
+            "S144m",
+            "S72",
+            "S144",
+            "S192",
+            "S192m",
+            "S192xm",
+            "S96",
+            "S112",
+            "S224",
+            "S224m",
+            "S224om",
+            "S224oo",
+            "S224oom",
+            "S224ooo",
+            "S224se",
+            "S384",
+            "S384m",
+            "S384xm",
+            "S384xxm",
+            "S448",
+            "S448m",
+            "S448om",
+            "S448oo",
+            "S448oom",
+            "S448ooo",
+            "S448se",
+            "S576m",
+            "S576xm",
+            "S672",
+            "S672m",
+            "S672om",
+            "S672oo",
+            "S672oom",
+            "S672ooo",
+            "S768",
+            "S768m",
+            "S768xm",
+            "S896",
+            "S896m",
+            "S896om",
+            "S896oo",
+            "S896oom",
+            "S896ooo",
+            "S960m"
+          ],
+          "x-ms-enum": {
+            "name": "azureLargeInstanceSizeNamesEnum",
+            "modelAsString": true
+          },
+          "description": "Specifies the Azure Large Instance SKU."
+        }
+      },
+      "description": "Specifies the hardware settings for the Azure Large Instance."
+    },
+    "Disk": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The disk name."
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Specifies the size of an empty data disk in gigabytes."
+        },
+        "lun": {
```

Comment:
I don't see the lun porperty inside the[ Disk struct ](https://msazure.visualstudio.com/One/_git/Compute-AzLinux-HanaRp/pullrequest/8477573?path=/src/AzureLargeInstance/apiversions/v20230720/types.go&_a=files). Please consider removing it.